src/CoreFoundation/CFArray.cs

```diff
@@ -38,54 +38,26 @@
 using CFArrayRef = System.IntPtr;
 using CFAllocatorRef = System.IntPtr;
 
+#nullable enable
+
 namespace CoreFoundation {
 	
-	partial class CFArray : INativeObject, IDisposable {
-
-		internal IntPtr handle;
+	public partial class CFArray : NativeObject {
 
 		internal CFArray (IntPtr handle)
-			: this (handle, false)
+			: base (handle, false)
 		{
 		}
 
 		[Preserve (Conditional = true)]
 		internal CFArray (IntPtr handle, bool owns)
+			: base (handle, owns)
 		{
-			if (handle == IntPtr.Zero)
-				throw new ArgumentNullException ("handle");
-
-			this.handle = handle;
-			if (!owns)
-				CFObject.CFRetain (handle);
 		}
 		
-		public IntPtr Handle {
-			get {return handle;}
-		}
-
 		[DllImport (Constants.CoreFoundationLibrary, EntryPoint="CFArrayGetTypeID")]
 		public extern static /* CFTypeID */ nint GetTypeID ();
```

Comment:
Do we want to expose this publicly?

Reply:
We expose them for other types... but they are not very useful.
I'll change it to `internal` like most others since it's how it effectively is today (without causing any harm)